react/react-data-fetching_iss-tracker/components/Controls/index.js

```diff
@@ -7,8 +7,8 @@ import {
 export default function Controls({ longitude, latitude, onRefresh }) {
   return (
     <ControlsContainer>
-      <ControlsDisplay>Lat: {longitude.toFixed(5)}</ControlsDisplay>
-      <ControlsDisplay>Long: {latitude.toFixed(5)}</ControlsDisplay>
+      <ControlsDisplay>Long: {longitude.toFixed(5)}</ControlsDisplay>
```

Comment:
you don't really need to change anything in the file. Look at my other comment :)